src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -86,7 +86,15 @@ export async function listItemGetController(req: Request, res: ListItemRes): Pro
   };
 
   const isPinned = listItem?.pinnedBy?.some((user) => userId === user.id) ?? false;
-  const actionButtonsForStatus = actionButtons[listItem.status];
+  let actionButtonsForStatus = actionButtons[listItem.status];
+
+  if (getPublishingStatus(listItem) !== PUBLISHING_STATUS.live) {
```

Comment:
getPublishingStatus is used twice, so it would be simpler to put it in a variable first. 

```
const publishingStatus = getPublishingStatus(listItem);
```

But I think this would be based on `listItem.isPublished`, rather than live. the live publishing status is if there's no action needed, and they are published. If they are in annual review mode (`listItem.isAnnualReview`), they won't be `live`.